spec/integ/matrix-client-sliding-sync.spec.js

```diff
@@ -0,0 +1,499 @@
+import { SlidingSync, SlidingSyncState } from "../../src/sliding-sync";
+import { TestClient } from "../TestClient";
+import { logger } from '../../src/logger';
+
+/**
+ * Tests for sliding sync. These tests are broken down into sub-tests which are reliant upon one another.
```

Comment:
huh - I would have thought jest would just split them up and run them across multiple workers, but they seem to have passed...